### DISCUSSION TITLE
İşimize yarayabilecek iki ufak proje

### DISCUSSION THREAD

**Discussion by AezeXOffical** [acikkaynak/deprem-yardim-projesi#74] [Ideas]
Selam, ilk proje çok basit aslında (Yapılıp yapılmadığını tam olarak bilmiyorum): 
QR kodlar ile sokaklar olsun farklı bölgeler, TV reklamlarında bulundurup insanların Applerimize girip ihbar, anlık durum görüntüleme yapabilir veya bağış yapılabilecek STK'lara doğrudan erişim sağlamasına imkan sunabiliriz. Bunun için işin dijital kısmını halletmek çok daha kolay. Eğer sokaklar veya bölgelere Sticker halinde dağıtım yapılacak ise fotoğraf-baskı ile uğraşan insanlara ulaşmak gerekiyor.

İkinci projem buna nazaran daha kapsamlı: 
Sahadaki ekiplerin eline ulaştırabileceğimiz yüz tanımlama sistemi. Benim hızlıca yazdığım ufak bir kod mevcut kişinin yüz hatlarını algılıyor. Ayrıca diğer dev. arkadaşlarda destek olursa, eklemeler yapıp bunu daha ileriye taşıyabiliriz. Maalesef afet bölgelerinde çocuk kaçırılmaları mevcut. Bu kötü durumun önüne bir nebze de olsa bu şekilde geçebiliriz. Enkaz altından çıkarılan çocukların yüz tanımaları yapılarak, kimin gerçek akraba veya tanıdıklarının olduğu görülebilir. Bu şekilde kötü niyetli kişilerin amaçlarının önüne çok etkili olmasa da geçmiş olabileceğimizi düşünüyorum. Bu proje için tahminimce KVKK 'nın izin verdiği ölçüde gerekli yerlere ulaşıp insanların mahremiyetine de çok girmeden bu fikri hayata geçirebileceğimize inanıyorum. Olumlu veya olumsuz geri dönüşlerinizi bekliyor olacağım, teşekkürler.

**Comment by sercangoger** [comment]
Canlı yayınlar yapan youtuber ve kanallar için QRCode mantıklı.

**Comment by sercangoger** [comment]
AI takımı cocuklar içn yüz tanımaya bakabilir
